src/Sylius/Behat/Context/Api/Shop/ProductVariantContext.php

```diff
@@ -274,6 +300,43 @@ public function variantAndVariantShouldBeVisible(array $variants): void
         }
     }
 
+    /**
+     * @Then /^I should see variant with ("[^"]+" option) and ("[^"]+" option value) priced at ("[^"]+") at (\d+) position$/
+     */
+    public function iShouldSeeVariantWithOptionPricedAtAtPosition(
+        ProductOptionInterface $option,
+        ProductOptionValueInterface $optionValue,
+        int $price,
+        int $position,
+    ): void {
+        $collection = $this->responseChecker->getCollection($this->client->getLastResponse());
+
+        Assert::same($collection[$position - 1]['price'], $price);
+        Assert::same(
+            $collection[$position - 1]['options'][self::FIRST_COLLECTION_ITEM]['option'],
+            $this->iriConverter->getIriFromItem($option),
+        );
+        Assert::same(
+            $collection[$position - 1]['options'][self::FIRST_COLLECTION_ITEM]['optionValue'],
+            $this->iriConverter->getIriFromItem($optionValue),
+        );
+    }
+
+    /**
+     * @Then /^I should not see variant with "([^"]+)" option "([^"]+)" priced at ("[^"]+")$/
+     */
+    public function iShouldNotSeeVariantWithOptionPricedAt(string $optionName, string $optionValue, int $price): void
+    {
+        $collection = $this->responseChecker->getCollection($this->client->getLastResponse());
+
+        foreach ($collection as $variant) {
+            if ($variant['price'] === $price) {
+                Assert::notContains($variant['options'][0], strtoupper($optionName));
+                Assert::notContains($variant['optionValues'][0], strtoupper($optionValue));
+            }
```

Comment:
Are these asserts correct?
From what I see, the `optionValues` node was removed from the shop's endpoint.

Reply:
Good catch, this piece of code was not even run because of how the test was written (different prices of variants). I updated the steps 🖖